src/decorators.js

```diff
@@ -58,7 +58,7 @@ export function customElement(name: string): any {
 * @param defaultBindingMode The default binding mode to use when the attribute is bound with .bind.
 * @param aliases The array of aliases to associate to the custom attribute.
 */
-export function customAttribute(name: string, defaultBindingMode?: number, aliases?: string[]): any {
+export function customAttribute(name: string, defaultBindingMode?: bindingMode, aliases?: string[]): any {
```

Comment:
`bindingMode` is unfortunately a const object at this time. So, I don't think it can be used as a type. Can we revert this back to number?

Reply:
Note that with newest API on `BindableProperty`, it now understands lookup property of `bindingMode` such as `toView`, `fromView`, `twoView`, `oneTime`

@EisenbergEffect do we need to adjust this based on that ?

Reply:
@bigopon So, it will accept either a string or a number?

Reply:
@EisenbergEffect Yes it's added in this commit https://github.com/aurelia/templating/commit/e7da9730eaafb6c9522a659fa8355f0f49188e56#diff-8fecf3f03eea2e940742fe22c0791bfbR51 

It was to support Aurelia plugins / classes